tools/arcilator/arcilator.cpp

```diff
@@ -59,6 +59,9 @@ static cl::opt<std::string> outputFilename("o", cl::desc("Output filename"),
                                            cl::init("-"),
                                            cl::cat(mainCategory));
 
+static cl::opt<bool> tap("tap", cl::desc("Tap all ports"), cl::init(false),
```

Comment:
Enabling the `tapWire` option would not be accessible here, right? Should we make this option an enum instead and pass the options to the pass?

Reply:
This is probably more like a set of flags that allows you to independently enable ports and wires. I'll just create two separate options for now. In the future we'll probably want to allow the user to specify certain modules in the hierarchy that they want to observe.